packages/editor-ui/src/components/SettingsSidebar.vue

```diff
@@ -19,6 +19,12 @@
 				</i>
 				<span slot="title">{{ $locale.baseText('settings.users') }}</span>
 			</n8n-menu-item>
+			<n8n-menu-item index="/settings/community-nodes" :class="$style.tab">
```

Comment:
if feature is behind a flag, we need to hide this

Reply:
Yup, updated.